demisto_sdk/commands/download/downloader.py

```diff
@@ -344,7 +353,9 @@ def build_custom_content_object(file_path: str) -> dict:
         """
         file_data, file_ending = get_dict_from_file(file_path)  # For example: yml, for integration files
         file_type: str = find_type(_dict=file_data, file_type=file_ending)  # For example: integration
-        file_entity: str = ENTITY_TYPE_TO_DIR.get(file_type)  # For example: Integrations
+        file_entity = ENTITY_TYPE_TO_DIR.get(file_type)  # For example: Integrations
```

Comment:
why did you change this? whats returned from the func?

Reply:
ENTITY_TYPE_TO_DIR dict contains only strings, what is wrong here?

Reply:
it can return None.